libs/s25main/ingameWindows/iwTransport.cpp

```diff
@@ -53,42 +48,44 @@ iwTransport::iwTransport(const GameWorldViewer& gwv, GameCommandFactory& gcFacto
     // Buttons der einzelnen Waren anlegen
     ctrlOptionGroup* group = AddOptionGroup(6, ctrlOptionGroup::ILLUMINATE);
 
-    // Zeiger auf die Bilder für die einzelnen Waren in der Transportschlange
-    TRANSPORT_SPRITES[0] = LOADER.GetMapTexN(2250 + GD_COINS);
-    TRANSPORT_SPRITES[1] = LOADER.GetTextureN("io", 111);
-    TRANSPORT_SPRITES[2] = LOADER.GetMapTexN(2250 + GD_BEER);
-    TRANSPORT_SPRITES[3] = LOADER.GetMapTexN(2250 + GD_IRON);
-    TRANSPORT_SPRITES[4] = LOADER.GetMapTexN(2250 + GD_GOLD);
-    TRANSPORT_SPRITES[5] = LOADER.GetMapTexN(2250 + GD_IRONORE);
-    TRANSPORT_SPRITES[6] = LOADER.GetMapTexN(2250 + GD_COAL);
-    TRANSPORT_SPRITES[7] = LOADER.GetMapTexN(2250 + GD_BOARDS);
-    TRANSPORT_SPRITES[8] = LOADER.GetMapTexN(2250 + GD_STONES);
-    TRANSPORT_SPRITES[9] = LOADER.GetMapTexN(2250 + GD_WOOD);
-    TRANSPORT_SPRITES[10] = LOADER.GetMapTexN(2250 + GD_WATER);
-    TRANSPORT_SPRITES[11] = LOADER.GetTextureN("io", 80);
-    TRANSPORT_SPRITES[12] = LOADER.GetMapTexN(2250 + GD_HAMMER);
-    TRANSPORT_SPRITES[13] = LOADER.GetMapTexN(2250 + GD_BOAT);
-
+    buttonData = {{{LOADER.GetMapTexN(2250 + GD_COINS), WARE_NAMES[GD_COINS]},
```

Comment:
WARE_NAMES is still gettext_noop?

Reply:
Yes, see libs/s25main/gameTypes/GoodTypes.h

I just wanted to avoid copying all the texts here

Reply:
okay, only wanted to make sure that this is not the only place where these are translated :-D